crml/staking/src/lib.rs

```diff
@@ -771,6 +789,46 @@ decl_storage! {
 	}
 }
 
+fn migrate_payees_to_rewards_module<T: Trait>() -> frame_support::weights::Weight {
+	#[allow(dead_code)]
+	mod inner {
+		use codec::{Decode, Encode};
+		use sp_std::vec::Vec;
+		#[derive(Encode, Decode, Default)]
```

Comment:
nit: is it possible to import the real type definition here?

Reply:
Can you please explain a bit more?

Reply:
I have removed the old structure definition. The new definition is internally different from the old one and follows the latest as in substrate.